packages/cloud-functions/merkle-distributor-helper/index.js

```diff
@@ -0,0 +1,18 @@
+const { getClaimsForAddress } = require("@uma/merkle-distributor");
```

Comment:
Just curious -- why not typescript?

Reply:
Looks like Cloud functions _do_ support it. Additionally, the package `@google-cloud/functions-framework` contains the appropriate types for the exported methods. I avoided it initially as all the googling I did around it involves a fair amount _more_ setup, tsconfig, and a few other dependencies to get it to function 100%. We can try changing this in the future I think.

I also saw a few cool patterns around building more complex sets of serverless functions such as [this](https://github.com/flatfisher/cloud-functions-typescript-template) repo and [this](https://github.com/robisim74/firebase-functions-typescript-starter) repo.

Also, the functions-framework-nodejs (a package I used to run tests against the serverless functions locally) supports typescript. check it out [here](https://github.com/GoogleCloudPlatform/functions-framework-nodejs)

Reply:
In fact the functions-framework-nodejs has some very simple [getting started on using typescript](https://github.com/GoogleCloudPlatform/functions-framework-nodejs/blob/master/docs/typescript.md). would recommend you look at this @mrice32 before building the cashing engine for the price feeds we discussed IRL during spring planning.

Reply:
>  I avoided it initially as all the googling I did around it involves a fair amount more setup, tsconfig, and a few other dependencies to get it to function 100%. We can try changing this in the future I think.

Makes sense. This is a pretty new setup, so I totally get trying to reduce the number of unknowns. 

From what you said it sounds like it's highly doable, but agree that it's probably not worth it until after we get TVL options up and running and stable.